code/game/machinery/computer/manager_camera.dm

```diff
@@ -230,9 +257,17 @@ GLOBAL_VAR_INIT(execution_enabled, FALSE)
 			H.apply_status_effect(/datum/status_effect/interventionshield/black)
 		if(MANAGER_PALE_BULLET)
 			H.apply_status_effect(/datum/status_effect/interventionshield/pale)
+		if(MANAGER_QUAD_BULLET)
+			if (H.has_status_effect(/datum/status_effect/interventionshield))
+				playsound(get_turf(src), 'sound/weapons/empty.ogg', 10, 0, 3)
+				to_chat(owner, span_warning("ERROR: TARGET HAS A SHIELD ALREADY."))
+				return FALSE
```

Comment:
This feels like it'd confuse people due to this being the only bullet that has the warning, something should be done to make this consistent